src/Views/scstormrunoff.html

```diff
@@ -135,66 +134,125 @@ <h4>Citation</h4>
                                     </div>
                                 </div>
                                 <div class="form-row">
                                     <div class="form-column-left">
-                                        <label for="PeakRateFactor">Peak Rate Factor, PRF:</label> <span ng-if="SyntheticUrbanHydrograph.PeakRateFactor.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
-                                        <input type="number" name="PeakRateFactor" ng-model="vm.peakRateFactor" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                        <label for="WatershedRetention">Watershed Retention, S:</label> <span ng-if="SyntheticUrbanHydrograph.WatershedRetention.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                        <input type="number" name="WatershedRetention" ng-model="vm.watershedRetention" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required readonly/>
                                     </div>
                                     <div class="form-column-right">
-                                        <label for="StandardCurveNumber">Standard Curve Number, CN:</label> <span ng-if="SyntheticUrbanHydrograph.StandardCurveNumber.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
-                                        <input type="number" name="StandardCurveNumber" ng-model="vm.standardCurveNumber" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                        
                                     </div>
                                 </div>
-                                <div class="form-row">
-                                    <div class="form-column-left">
-                                        <label for="WatershedRetention">Watershed Retention, S:</label> <span ng-if="SyntheticUrbanHydrograph.WatershedRetention.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
-                                        <input type="number" name="WatershedRetention" ng-model="vm.watershedRetention" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+
+
+                                <!-- Curve Number -->
+                                <h3 class="center-div">Curve Number</h3>
+                                <div class="wim-alert hidden-print">Select a 'Standard Curve Number Value' and a 'Curve Number Modification Method', once selected 'Standard Curve Number' will be calculated with <b>Calculate Missing Parameters</b>.</div>
```

Comment:
Grammar changes: `Select a 'Standard Curve Number Value' and a 'Curve Number Modification Method'. Once selected, 'Standard Curve Number' will be calculated with <b>Calculate Missing Parameters</b>.</div>`